pkgs/tools/misc/xppen/default.nix

```
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchurl
+, libusb
+, autoPatchelfHook
+, qt5
+, libGL
+}:
+stdenv.mkDerivation rec {
+  pname = "xppen";
+  version = "3.4.9";
+
+  src = fetchurl {
+    url = "https://www.xp-pen.com/download/file.html?id=1936&pid=990&ext=gz";
+    hash = "sha256-33B8JjOjsv17uFae3zhGKw40oeD6fHJ1YC9wEVQO7+Q=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [ qt5.full libusb libGL ];
+
+  dontConfigure = true;
+  dontBuild = true;
+  dontCheck = true;
+
+  unpackPhase = ''
+        runHook preUnpack
+        tar xzvf ${src} --strip-components=1
+        runHook postUnpack
+  '';
```

Is this `unpackPhase` set manually because `fetchurl` doesn't recognize the extension? In that case, do you think you might be able to get away with using `fetchzip` and the `extension` flag instead? (the hash might need to be recalculated)

```suggestion
  src = fetchzip {
    url = "https://www.xp-pen.com/download/file.html?id=1936&pid=990&ext=gz";
    hash = "sha256-33B8JjOjsv17uFae3zhGKw40oeD6fHJ1YC9wEVQO7+Q=";
    extension = ".tar.gz";
  };

  nativeBuildInputs = [ autoPatchelfHook ];

  buildInputs = [ qt5.full libusb libGL ];

  dontConfigure = true;
  dontBuild = true;
  dontCheck = true;
```